packages/react-composites/src/composites/ChatComposite/FileDownloadCards.tsx

```diff
@@ -39,7 +42,21 @@ export interface FileDownloadCards {
 export const FileDownloadCards = (props: FileDownloadCards): JSX.Element => {
   const truncateLength = 15;
   const { userId, message } = props;
+  // @conditional-compile-remove(file-sharing)
+  const [showSpinner, setSpinner] = useState(false);
   const fileDownloads: FileMetadata[] = message.metadata ? extractFileMetadata(message.metadata) : [];
+  const DownloadIconTrampoline = (): JSX.Element => {
```

Comment:
This component can be declared outside the `FileDownloadCards` component to clean up the code a bit.

Reply:
However, I think this component is not needed. An inline logic looks much simpler for actionIcon
`actionIcon={showSpinner ? <Spinner size={SpinnerSize.medium} aria-live={'assertive'} /> : <ChatCompositeIcon iconName="Download" /> }`

Reply:
Done